purebred.spec

```diff
@@ -19,58 +19,77 @@ BuildRequires:  ghc-Cabal-devel
 BuildRequires:  ghc-rpm-macros
 # Begin cabal-rpm deps:
 BuildRequires:  chrpath
+BuildRequires:  ghc-attoparsec-devel
 BuildRequires:  ghc-brick-devel
 BuildRequires:  ghc-bytestring-devel
 BuildRequires:  ghc-case-insensitive-devel
 BuildRequires:  ghc-directory-devel
-BuildRequires:  ghc-exceptions-devel
 BuildRequires:  ghc-filepath-devel
 BuildRequires:  ghc-lens-devel
-BuildRequires:  ghc-mime-devel
 BuildRequires:  ghc-mime-mail-devel
 BuildRequires:  ghc-mtl-devel
 BuildRequires:  ghc-network-devel
 BuildRequires:  ghc-notmuch-devel
 BuildRequires:  ghc-optparse-applicative-devel
 BuildRequires:  ghc-process-devel
+BuildRequires:  ghc-purebred-email-devel
 BuildRequires:  ghc-text-devel
 BuildRequires:  ghc-text-zipper-devel
 BuildRequires:  ghc-time-devel
-BuildRequires:  ghc-unix-devel
 BuildRequires:  ghc-vector-devel
 BuildRequires:  ghc-vty-devel
 BuildRequires:  ghc-xdg-basedir-devel
 %if %{with tests}
 BuildRequires:  ghc-ini-devel
-BuildRequires:  ghc-quickcheck-text-devel
+BuildRequires:  ghc-quickcheck-instances-devel
 BuildRequires:  ghc-regex-posix-devel
-BuildRequires:  ghc-resourcet-devel
 BuildRequires:  ghc-tasty-devel
 BuildRequires:  ghc-tasty-hunit-devel
 BuildRequires:  ghc-tasty-quickcheck-devel
 BuildRequires:  ghc-temporary-devel
-BuildRequires:  ghc-unbounded-delays-devel
 %endif
+Requires: purebred-basic = %{version}-%{release}
+Requires: purebred-config = %{version}-%{release}
 # End cabal-rpm deps
 
 %description
 An mail user agent built around notmuch.
 
 
-%package devel
-Summary:        Haskell %{pkg_name} library development files
-Provides:       %{name}-static = %{version}-%{release}
+%package -n ghc-%{name}
+Summary:        Haskell %{name} library
+
+%description -n ghc-%{name}
+This package provides the Haskell %{name} shared library.
+
+
+%package -n ghc-%{name}-devel
+Summary:        Haskell %{name} library development files
+Provides:       ghc-%{name}-static = %{version}-%{release}
 Requires:       ghc-compiler = %{ghc_version}
 Requires(post): ghc-compiler = %{ghc_version}
 Requires(postun): ghc-compiler = %{ghc_version}
-Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       ghc-%{name}%{?_isa} = %{version}-%{release}
+
+%description -n ghc-%{name}-devel
+This package provides the Haskell %{name} library development files.
+
+%package basic
```

Comment:
I would have thought `purebred-base` is a more idiomatic package name.
But could we even make it just `purebred`?

Reply:
The 'user installable' package is called purebred. The library development files are shipped as ghc-purebred-devel. I want to stay consistent with the [packaging guidelines](https://fedoraproject.org/wiki/Packaging:Haskell#Package_Naming) which, for libraries suggest to prefix with `ghc-`.

Reply:
In regards to `basic` vs `base` ... no hard feelings and I'm happy to rename it to `base`. Actually sounds better ...

Reply:
Ah.. one more thing: I picked `basic` because I looked at the xmonad package. I don't want to repeat their mistakes, but that's the reason I had chosen `basic`.